spec/controllers/idv/personal_key_controller_spec.rb

```diff
@@ -266,6 +362,16 @@ def index
         end
       end
     end
+
+    context 'personal key already acknowledged' do
```

Comment:
As noted above, I think this is a back button prevention check and it should just show the personal key rather than redirecting. Especially since the user might realize they didn't save their key after all and want to go back and look at it.

Reply:
Yeah, I think you may be right

Reply:
Thinking more about it, this would be a change from how we operate right now--we explicitly remove the personal key from the session after it's been acknowledged. So I think showing the basic "Your identity has already been verified" for now is the right call and I can capture the change in a follow up ticket

Reply:
Did you have a chance to put in this followup ticket?